Frontend/src/pages/Country.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { useParams, useNavigate } from "react-router-dom";
+
+import styled from "styled-components";
+import { FaSortDown } from "react-icons/fa";
+
+import AttractionCards from "components/AttractionCards";
+import MapLocation from "components/MapLocation";
+
+const Country = () => {
+  const { country } = useParams();
+  const [visible, setVisible] = useState(true);
+  const [category, setCategory] = useState("");
+  const [land, setLand] = useState(country); //land takes the place of country in useParams
+
+  const navigate = useNavigate();
+
+  let attractions = useSelector((store) =>
+    store.sightseeing.sightseeings.filter((item) => item.country === land)
+  );
+
+  const categoryAttractions = attractions.filter(
+    (item) => item.category === category
+  );
+
+  const showMenu = () => {
+    if (visible) {
+      setVisible(false);
+    } else {
+      setVisible(true);
+    }
+  };
+  const categories = ["food", "culture", "activity", "music"];
+  const countries = ["Sweden", "Norway", "Denmark"];
+
+  const handleCategory = (event) => {
+    setCategory(event.target.value);
+  };
+  // FOR ACTIVITY PAGE "ADD A COMMENT"
+
+  return (
+    <>
+      <Main>
+        <FilteringContainer>
+          <h2 style={{ marginLeft: 15 }}>{land}</h2>
+          <StyledToggle onClick={showMenu}>
+            <FilterText>
+              Filter Menu <FaSortDown />
+            </FilterText>
+          </StyledToggle>
+          {visible && (
+            <ToggleContainer>
+              <CountryChooseContainer>
+                <FilterText>Country</FilterText>
+                <form>
+                  <label htmlFor="country"></label>
+                  <Select
+                    id="country"
+                    value={land}
+                    onChange={(event) => {
+                      setLand(event.target.value); //asynchronous
+                      navigate(`/country/${event.target.value}`);
+                      setCategory("");
+                    }}
```

Comment:
It's a little bit too much of code to put directly in JSX - would be great to create separate function and pass it here